src/js/options.js

```diff
@@ -111,3 +113,13 @@ restore.addEventListener("click", ()=>{
         });
     }
 });
+
+
+
+localimage.addEventListener("change",()=>{
+    
+    var tmppath = URL.createObjectURL(localimage.files[0]);
+    localStorage.setItem("theValue",tmppath);
+    alert("Image Updated");
+  
+});
```

Comment:
Add EOF. Thanks :-)

Reply:
@Akshat-Jain  What I have to write to mention EOF??

Reply:
Done

Reply:
@gabru-md  Please merge it.

Reply:
Can you add an empty line before EOF

Reply:
Ok